src/src/pages/capabilities/details.js

```diff
@@ -33,20 +32,17 @@ function CapabilityDetailsPageContent() {
     name,
     kafkaClusters,
     loadCapability,
-    showResources,
     showCosts,
     isPendingDeletion,
     isDeleted,
     updateDeletionStatus,
     awsAccount,
-    addNewInvitees,
-    isInviteesCreated,
   } = useContext(SelectedCapabilityContext);
 
   useEffect(() => {
-    window.scrollTo(0, 0);
+    window.scrollTo({ top: 0, behavior: "smooth" });
     loadCapability(id);
-  }, [id]);
+  }, [id, loadCapability]);
```

Comment:
updating on `loadcapability` here makes the page scroll to the top again all the time

Reply:
Good catch

I removed the scrolling; It should not affect anything.
Alternatively we make the loadCapability a useCallback, but that feels wrong-er 🤷